AcademyApp/ViewControllers/LoginViewController.swift

```diff
@@ -80,151 +85,131 @@ extension LoginViewController : UITextFieldDelegate
+            as? NSValue)?.cgRectValue  else {
             return
         }
-        
-        //adjusting vertical spacing of elements on login screen based on whether the keyboard is visible
-        
+
+        // adjusting vertical spacing of elements on login screen based on whether the keyboard is visible
+
         scrollViewBottomConstraint.constant = isVisible ?  -keyboardRect.height  : 0
-        doneButtonLoginFormVerticalSpacingConstraint.constant = isVisible ? Config.adjustedDoneButtonLoginFormVerticalSpacing : defaultDoneButtonLoginFormVerticalSpacingConstant
-        
-        //call layoutIfNeeded() before scroll() otherwise aniamtion will break
-        
+        verticalSpacingConstraint.constant = isVisible ? Config.adjustedVerticalSpacing : defaultVerticalSpacingConstant
+
+        // call layoutIfNeeded() before scrollView.scrollTo() otherwise animation will break
+
         self.view.layoutIfNeeded()
         scrollView.scrollTo(direction: .bottom, animated: false)
     }
 }
 
-
 // MARK: - Setup Methods
 
-private extension LoginViewController
-{
-    var textFields : [UITextField] {
-        return [emailTextField, passwordTextField ]
-    }
-    
+private extension LoginViewController {
+
     func setup() {
         setupUI()
         setDelegates()
         addObservers()
     }
-    
+
     // MARK: - Notification Center Setup
-    
-    func addObservers(){
-        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(notification:)), name: UIResponder.keyboardWillShowNotification, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide(notification:)), name: UIResponder.keyboardWillHideNotification, object: nil)
+
+    func addObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(notification:)),
+                                               name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide(notification:)),
+                                               name: UIResponder.keyboardWillHideNotification, object: nil)
     }
-    
-    func removeObservers(){
+
+    func removeObservers() {
         NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
         NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
     }
-    
+
     // MARK: - Delegation setup
-    
-    func setDelegates(){
+
+    func setDelegates() {
         emailTextField.delegate = self
         passwordTextField.delegate = self
     }
-    
+
     // MARK: - UI Setup Methods
-    
-    func setupUI(){
+
+    func setupUI() {
         backgroundColorSetup()
         textFieldsSetup()
-        showLabelSetup()
+        showButtonSetup()
```

Comment:
would suggest have different name... its more like setup some kind of button, showButtonSetup sounds like func will show setup for button

Reply:
Renamed all setup methods to setup_______()